SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1873,126 +1900,36 @@ void CNEMOEulerSolver::SetNondimensionalization(CConfig *config, unsigned short
 
 void CNEMOEulerSolver::BC_Sym_Plane(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics,
                                     CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
-
-  unsigned short iDim, jDim, iSpecies, iVar, jVar;
   unsigned long iPoint, iVertex;
+  unsigned short iDim, iVar;
+  su2double Area, Normal[3], UnitNormal[3];
+  const su2double* Residual_Old;
+  su2double *Residual = new su2double[nVar];
```

Comment:
MAXNDIM and if you want you can make Residual a static array of size MAXNVAR, more practical and efficient than new delete everytime.